src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -106,7 +107,15 @@ internal partial class Binder
 
                     foreach (var attributeList in p.AttributeLists)
                     {
-                        Error(diagnostics, ErrorCode.ERR_AttributesNotAllowed, attributeList);
+                        if (syntax.Kind() == SyntaxKind.ParenthesizedLambdaExpression)
+                        {
+                            // PROTOTYPE: Report error if parameter syntax is missing the type (using implicit parameter type syntax).
+                            MessageID.IDS_FeatureLambdaAttributes.CheckFeatureAvailability(diagnostics, attributeList);
```

Comment:
Do we do a similar check on attribute on the lambda too?

Reply:
You may have missed this question (should we only use semantic checks for LangVer checks?)
Could you also check if IDS_FeatureLambdaAttributes has the appropriate comment (syntax vs. semantic check)?


---
In reply to: [607473773](https://github.com/dotnet/roslyn/pull/52178#discussion_r607473773) [](ancestors = 607473773)